cppcheckSuppressions.txt

```diff
@@ -0,0 +1,38 @@
+*:3rdParty/*
+
+ConfigurationNotChecked:*
+ctuOneDefinitionRuleViolation:Examples/*
+missingInclude:*
+unknownMacro:*
+unusedFunction:*
+unusedStructMember:*
+
+constParameter:Common++/header/SystemUtils.h:309
+duplInheritedMember:Packet++/header/SomeIpSdLayer.h:199
+duplInheritedMember:Packet++/header/SomeIpSdLayer.h:275
+knownConditionTrueFalse:Tests/Pcap++Test/main.cpp:148
+knownConditionTrueFalse:Examples/Tutorials/Tutorial-DpdkL2Fwd/WorkerThread.cpp:19
+knownConditionTrueFalse:Examples/DpdkExample-FilterTraffic/AppWorkerThread.h:73
+knownConditionTrueFalse:Tests/Packet++Test/main.cpp:90
+localMutex:Pcap++/src/NetworkUtils.cpp:171
+localMutex:Pcap++/src/NetworkUtils.cpp:420
+moduloofone:Examples/DpdkBridge/main.cpp:371
+shiftNegative:Pcap++/src/PfRingDevice.cpp:609
+shiftNegative:Pcap++/src/PfRingDevice.cpp:721
+uninitdata:Pcap++/src/PcapLiveDevice.cpp:870
+uninitMemberVar:Packet++/header/IcmpLayer.h:399
+unreadVariable:Packet++/src/DnsResource.cpp:111
+unreadVariable:Packet++/src/DnsResource.cpp:137
+unusedVariable:Tests/Pcap++Test/Tests/LoggerTests.cpp:168
+unusedVariable:Tests/Pcap++Test/Tests/LoggerTests.cpp:196
+unusedVariable:Tests/Packet++Test/Tests/SomeIpTests.cpp:23
+unusedVariable:Tests/Packet++Test/Tests/SomeIpTests.cpp:46
+unusedVariable:Tests/Packet++Test/Tests/SomeIpTests.cpp:173
+useStlAlgorithm:Tests/Pcap++Test/Tests/LoggerTests.cpp:187
```

Comment:
For the line-specific suppressions, maybe we should use `--inline-suppr` and put them inline?